pyquil/api/_abstract_compiler.py

```diff
@@ -130,14 +130,18 @@ def quil_to_native_quil(self, program: Program, *, protoquil: Optional[bool] = N
         # the event loop is available. Wrapping it in a Python async function ensures that
         # the event loop is available. This is a limitation of pyo3:
         # https://pyo3.rs/v0.17.1/ecosystem/async-await.html#a-note-about-asynciorun
-        async def _compile(*args) -> str:  # type: ignore
-            return await qcs_sdk.compile(*args)
+        async def _compile(*args, **kwargs) -> str:  # type: ignore
+            return await qcs_sdk.compile(*args, **kwargs)
 
         # TODO This ISA isn't always going to be available. Specifically, if the quantum processor is
```

Comment:
I know this isn't from this changeset, but is there a tracking issue?

Reply:
Good eye, yes, there is. It relates to the work that'll have to be done for #1496 - I updated the TODO.